lang/en/tool_mergeusers.php

```diff
@@ -186,8 +186,14 @@
     data from new user (by default). This also means that data from old user is 
     deleted to keep the consistence. Otherwise, if you uncheck this option, 
     data from old user will be kept.';
-
 $string['starttime'] = 'Started merging at {$a}';
 $string['finishtime'] = 'Finished merging at {$a}';
 $string['timetaken'] = 'Merge took {$a} seconds';
 $string['privacy:metadata'] = 'The Merge User Accounts plugin does not store any personal data.';
+$string['queue_to_process_pending_merge_requests'] = 'Get queue of the pending merge requests and launch adhoc task for each single request';
+$string['cannotfinduser'] = 'Cannot find the user with {$a->userfield} {$a->uservalue}. Change the input parameters.';
```

Comment:
added as requested. In the calling function get_string has changed as follow:
 throw new moodle_exception(get_string('cannotfinduser', 'tool_mergeusers',
                                        (object)['userfield' => $userfield, 'uservalue' => $uservalue]));

Reply:
fantastic!